src/solver/conv_winoRxS.cpp

```diff
@@ -191,31 +191,40 @@ inline bool IsShaderContraintsMet(const int R,
              params.problem.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = params.GetStream().GetMaxHardwareComputeUnits();
     if(!params.problem.IsLayoutDefault())
     {
         return false;
     }
 
+    uint32_t o_K_stride      = OH * OW;
+    uint32_t o_N_stride      = o_K_stride * K;
+    uint32_t o_N_stride_OHOW = o_N_stride + o_K_stride;
+
+    uint32_t d_C_stride    = H * W;
+    uint32_t d_N_stride    = d_C_stride * C;
+    uint32_t d_N_stride_HW = d_N_stride + d_C_stride;
+
+    auto num_tiles  = Ceil(OH, 2) * Ceil(OW, 2);
+    auto stride_one = (params.problem.in_stride == 1 && params.problem.out_stride == 1);
```

Comment:
I'm not sure if this is the correct way to check stride 1 (see [ProblemDescriptor](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/src/include/miopen/problem_description.hpp#L90-L95)). What happens if `-u` `-v` are different?

Reply:
Thanks, fixed in e06943b !